examples/apps/view-framework-sampler/src/app.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { getTinyliciousContainer } from "@fluid-experimental/get-container";
+import { getTinyliciousContainer } from "@fluid-experimental/tinylicious-client";
```

Comment:
I'm thinking it will be better to isolate this change to adding `tinylicious-client` and updating the FluidStatic uses cases. Which is just the one app-container example.

We definitely have a todo to update all our examples to the new model.

Reply:
Removed all the changes to other consuming packages other than `app-integration/external-controller`.
Created https://github.com/microsoft/FluidFramework/issues/5853 to track and added a note pointing to it in get-container's README